tw/pkg/commands/helm/testdata/helm-inventory/oci-repo.txtar

```diff
@@ -0,0 +1,5 @@
+helm-inventory --path inventory.json -- helm template release oci://ghcr.io/stefanprodan/charts/podinfo --version 6.3.5 --namespace default --dry-run
```

Comment:
these require the network and will break if 6.3.5 ever changes, but in stefan we trust?